spread.yaml

```diff
@@ -84,6 +84,8 @@ backends:
                 workers: 4
```

Comment:
Should we remove  - opensuse-42.3-64: from above or may this come back eventually?

Reply:
I would keep it until its EOL later this year.